pgm_learner/LinearGaussianNodeState[] node_states
